packages/ui-checkbox/src/CheckboxGroup/__tests__/CheckboxGroup.test.tsx

```diff
@@ -102,7 +102,9 @@ describe('<CheckboxGroup />', async () => {
     )
 
     expect(consoleError).to.have.been.calledWithMatch(
-      `Failed prop type: You provided a 'value' prop without an 'onChange' handler on 'CheckboxGroup'`
+      match.string,
+      match.string,
+      `You provided a 'value' prop without an 'onChange' handler on 'CheckboxGroup'`
```

Comment:
This is another, less worrying change: The PropType validators now call console.error with an array of arguments and use printf-like formatting now, so we need to match the 3rd argument of `console`